src/platform/user/tests/authentication/utilities.unit.spec.js

```diff
@@ -269,7 +270,7 @@ describe('Authentication Utilities', () => {
     });
 
     it('should NOT return session url with _verified appended for external applications other than OCC/Flagship', () => {
-      setup({ path: usipPathWithParams(mhvUsipParams) });
+      setup({ path: usipPathWithParams(ebenefitsUsipParams) });
```

Comment:
why are you removing mhvusipparams here and replacing it with ebenefits?

Reply:
@joeniquette because it is the only USiP client that does not require a verified session.